scripts/init-mongo.ts

```diff
@@ -0,0 +1,92 @@
+import * as path from "path";
+import { promises as fs } from "fs";
+import mongoose, { isValidObjectId } from "mongoose";
+import {
+  ExpensesListModel,
+  ExpensesListDocument,
+} from "../src/features/expenses-list/expenses-list.model";
+import {
+  ExpensesModel,
+  Expense,
+} from "../src/features/expenses/expenses.model";
+import { UserModel } from "../src/features/users/user.model";
+
+const importData = async () => {
+  try {
+    await mongoose.connect("mongodb://localhost:27017/expanses-01");
```

Comment:
please use .env so we can work on-prem / cloud

Reply:
done